src/service/event.ts

```diff
@@ -0,0 +1,122 @@
+// External Dependencies
+import * as axios from 'axios'
+
+// Client
+import ClubHubClient from '../client'
+
+// Local Namespace.
+import Event from 'src/models/event'
+import User from 'src/models/user'
+
+/**
+ * Interface to the ClubHub `Event` API.
+ */
+export default class EventService {
+
+  /**
+   * The `ClubHubClient` instance for the service. 
+   */
+  public client: ClubHubClient
+
+  constructor(client: ClubHubClient) {
+    this.client = client
+  }
+
+  /**
+   * `GET` all the Events for the given calendar groupID.
+   */
+  public getEvents = async (groupID?: string, limit?: string, offset?: string, start?: string, end?: string): Promise<Event.Response> => {
```

Comment:
We might want to think about changing the parameters to be an object, which has all of these existing parameters as optional keys. It makes it way easier to deal with everything being optional, and easier to add new fields as we go